version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.0.22-SNAPSHOT"
+version in ThisBuild := "0.0.23-SNAPSHOT"
```

Comment:
Does this version not get bumped automatically? Or am I confusing it with the GraphQL library?

Reply:
It does usually but I deployed a version outside of Jenkins for reasons I can't remember so it needs a manual one.